contract_api/lambda_handler.py

```diff
@@ -48,6 +49,11 @@ def request_handler(event, context):
             org_id = sub_path[2]
             response_data = obj_reg.get_all_group_for_org_id(org_id=org_id)
 
+        elif re.match("(\/org\/)[^\/]*(\/group\/)[^\/]*[/]{0,1}$", path):
+            org_id = sub_path[2]
+            group_id = unquote(sub_path[4])
```

Comment:
wherever regular expression are being used give example what kind of request it will accept.